data/events/scripts/player.lua

```diff
@@ -315,3 +315,20 @@ function Player:onInventoryUpdate(item, slot, equip)
 		EventCallback.onInventoryUpdate(self, item, slot, equip)
 	end
 end
+
+function Player:onMinimapQuery(position)
+	if EventCallback.onMinimapQuery then
+		EventCallback.onMinimapQuery(self, position)
+	end
+
+	if not self:getGroup():getAccess() then
+		return
+	end
+
+	local tile = Tile(position)
+	if not tile then
+		Game.createTile(position)
```

Comment:
On thought, it doesn't seem correct create a tile...
```suggestion
		"warning message: unwalkable/null tile
		return
```

Reply:
I like the idea of being able to teleport to a non-existent area, as it can be close to where I want to go. And the fact of creating a floor for it is no problem as it will be removed in the next shutdown.

Reply:
I made /a, /up, /down and this feature possible to jump to empty positions in my fork because I didn't like how limiting they were. You rarely jump on empty tiles on production anyway.

Reply:
add local script setting to enable or throw error if walkable/null or not